### PR TITLE
Use BigDecimal for decimal numbers in the Integration API

### PR DESCRIPTION
**Pull request by grncdr**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -6,5 +6,6 @@ build:
     - TALONONE_PASSWORD=$$TALONONE_PASSWORD
   commands:
     - apk update
-    - apk add git ca-certificates ruby-json ruby-minitest
+    - apk add git ca-certificates ruby-json ruby-minitest ruby-bundler ruby-dev build-base
+    - bundle
```
Comment (janv) [inline .drone.yml]: should be `bundle install --deployment`

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 talon_one-*.gem
 secrets.yml
+Gemfile.lock
```
Comment (janv) [inline .gitignore]: Gemfile.lock needs to be checked in

**Comment by janv** [comment]
Uh, wait
There is no Gemfile
We don't need bundler at all

**Comment by janv** [comment]
Oh there is
nevermind

File: .gitignore (hunk 2 above)
Comment (grncdr) [inline .gitignore]: The [internet disagrees](https://www.google.de/search?q=checking+in+gemfile.lock)

**Comment by grncdr** [comment]
(discussed offline) we'll follow the recommendations of `bundle gem` for now.

**Pull request merged**